### PR TITLE
add some help for cog on readme

### PR DESCRIPTION
**Pull request by khancyr**

### CODE REVIEW

File: mavros/README.md [hunk 1]
```diff
@@ -255,6 +255,17 @@ Contributing
  - with `catkin tools`, issue `catkin run_tests`;
 11. If everything goes as planned, push the changes (`git push -u <remote_repo> <feature_branch>`) and issue a pull request.
 
+NOTE: cog.py can be use to generate/update enum from mavlink.
+To install it :
+
+    sudo pip install --user cogapp
```
Comment (vooon) [inline mavros/README.md]: You don't need sudo when use `--user` (module will be installed into `$HOME/.local/lib/python<ver>/...`).

**Comment by vooon** [comment]
Perhaps we need to separate contribution guide from readme.
And there we may be more specific about tools used.

File: mavros/README.md (hunk 1 above)
Comment (khancyr) [reply mavros/README.md]: yep right!

**Comment by khancyr** [comment]
I think that having a separated contrib.md could be an excellent idea!

**Pull request merged**